src/conference.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * ComfyChair
+ * Copyright (C) 2024, M. Venturino, G. Valenzuela
+ * July 20, 2024.
+ *
+ * MIT License
+ */
+
+#include "conference.hpp"
+#include "trackFactory.hpp"
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+
+Conference::Conference(const nlohmann::json& conferenceJson)
+{
+    if (conferenceJson.contains("chairs"))
+    {
+        for (const auto& chairJson : conferenceJson["chairs"])
+        {
+            m_chairs.push_back(std::make_shared<User>(chairJson));
+        }
+    }
+    if (conferenceJson.contains("authors"))
+    {
+        for (const auto& authorJson : conferenceJson["authors"])
+        {
+            m_authors.push_back(std::make_shared<User>(authorJson));
+        }
+    }
+    if (conferenceJson.contains("tracks"))
+    {
+        for (const auto& trackJson : conferenceJson["tracks"])
+        {
+            m_tracks.push_back(TrackFactory::createTrack(trackJson, TrackType::Regular));
```

Comment:
We need to use the JSON to check which type of Track we're going to create. This could be another issue 😃